test/tests/functional/pbs_provisioning.py

```diff
@@ -51,13 +51,16 @@
 e = pbs.event()
 vnode = e.vnode
 aoe = e.aoe
+if aoe == 'App1':
+    pbs.logmsg(pbs.LOG_DEBUG, "fake calling application provisioning script")
+    e.accept(1)
 pbs.logmsg(pbs.LOG_DEBUG, "aoe=%s,vnode=%s" % (aoe,vnode))
-pbs.logmsg(pbs.LOG_DEBUG, "fake calling os provisiong script")
+pbs.logmsg(pbs.LOG_DEBUG, "fake calling os provisioning script")
 e.accept(0)
 """
 
 
-class TestProvisioningJobWithHook(PBSTestSuite):
+class TestProvisioningJobWithHook(TestFunctional):
```

Comment:
Hi @riyazhakki ,
   Can you please change the class name as TestProvisioiningJob.

Warm regards
Dilip

Reply:
Hi @dilip-krishnan  
Incorporated your comment.